pages/project/read/views/index.jsx

```diff
@@ -63,7 +63,7 @@ function CurrentVersion({ model }) {
 function OpenTask({ model }) {
   const { openTask, editable, canUpdate, asruUser } = useSelector(state => state.static);
 
-  if (!openTask) {
+  if (!openTask || !openTask.data) {
```

Comment:
What case is this catching?

Reply:
If workflow is down, `openTask` is an object that looks like `{ action: 'failed' }` so there's no `data` prop and the use of it below causes the page to error rather than just skipping the stuff that needs the tasks. It might never happen outside of local env when you forget to start workflow.

Reply:
Makes sense. Thanks.